algo-dev-demo/digits-recognition/README.md

```diff
@@ -0,0 +1,104 @@
+# Digit recognition demo
+
+In this demo we'll go through deploying a pre-trained [scikit-learn](https://scikit-learn.org/) model for hand written digits recognition and we will explore some options to query this model to make inferences.
+
+The trained model is based on the [scikit-learn documentation](https://scikit-learn.org/stable/auto_examples/classification/plot_digits_classification.html).
+
+This demo features:
+
+- Hosting models in the Algorithmia platform
+- Calling other algorithms as part of the inference pipeline, on this case for image preprocessing
+- Use of multiple query types, on this case: url and base64
```

Comment:
on -> in

Reply:
I actually tried so hard to catch it this time lol